tfhe/docs/getting_started/quick_start.md

```diff
@@ -64,98 +44,101 @@ fn main() {
 }
 ```
 
-#### Boolean example
+Default configuration for x86 Unix machines:
+```toml
+tfhe = { version = "0.3.0", features = ["integer", "x86_64-unix"]}
+```
+
+Other configurations can be found [here](../getting_started/installation.md).
```

Comment:
I think it's better to link to ../how_to/rust_configuration.md here

Reply:
both being split feels odd as some features to compile the project are split between both files